server-api/installation.md

```diff
@@ -37,10 +38,12 @@ $ docker-compose run --rm --service-ports api yarn run dev:debug
 
 And debug your app with [Chrome Dev Tools](chrome://inspect).
 
-### Configuration
+### Setting up the database seeder for local development (Configuration)
```

Comment:
We should keep the more general section title `Configuration`. You can configure everything, not only the database seeder with these files.

Reply:
I will remove 
```suggestion
 (Configuration)
```
from the title.Is that okay?

Reply:
sure

Reply:
Corrected